ciri/node/protocol/gossip.h

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __CIRI_NODE_PROTOCOL_GOSSIP_H__
+#define __CIRI_NODE_PROTOCOL_GOSSIP_H__
+
+#include "utlist.h"
+
+#include "ciri/node/network/endpoint.h"
+#include "common/errors.h"
+#include "common/trinary/bytes.h"
+#include "common/trinary/flex_trit.h"
+
+// The amount of bytes making up the non signature message fragment part of a transaction gossip payload
+#define GOSSIP_NON_SIG_BYTES_LENGTH 292
```

Comment:
should we call that `GOSSIP_ESSENCE_BYTES_LENGTH` instead?

Reply:
I agree we maybe find another name but technically it's not the essence, it's all the fields except the signature.

Reply:
If so, current name is the best